### DISCUSSION TITLE
Problemas para fazer o build no IOS após Atualização do Mac.

### DISCUSSION THREAD

**Discussion by Zeca-dev** [Flutterando/forum#145] [Perguntas e Respostas] [answered]
Após atualizar o Mac tive problemas para fazer o build para iOS pois estava apresentando o problema mostrado nessa imagem:

<img width="1228" alt="image" src="https://github.com/Flutterando/forum/assets/50033704/0593cc52-bf44-4f52-acdc-b283f34de357">

**Comment by Zeca-dev** [comment, marked as answer]
Consegui resolver recriando a pasta dentro da instalação do Xcode, conforme a resposta: 

https://stackoverflow.com/questions/75574268/missing-file-libarclite-iphoneos-a-xcode-14-3#:~:text=Open%20Terminal%20and%20go%20to%20the%20following%20folder%3A

**Reply by Zeca-dev** [reply]
Pessoal, eu postei essa dúvida aqui com a resposta para fins de registro e solução, pois acredito que mais pessoas podem estar passando por esse problema.

**Comment by emiliaferlin** [comment]
@Zeca-dev cara, tua solução me ajudou muito no flutter build ipa, mas quando vou dar flutter run, no caso para abrir no emulador, está me dando o seguinte erro, não sei se chegou a dar para você também...  Command CodeSign failed with a nonzero exit code
    Command SwiftCompile failed with a nonzero exit code
    warning: Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Runner' from project 'Runner')
    warning: Run script build phase 'Thin Binary' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Runner' from project 'Runner')


A tua solução me ajudou na build para as lojas, agora não tem jeito de abrir no emulador, grande abraço.

**Reply by emiliaferlin** [reply]
Se puder me ajudar nessa também...

**Reply by Zeca-dev** [reply]
Alguns desses erros eu resolvo fazendo flutter clean, flutter pub get, abrindo pelo Xcode e fazendo um build por ele. Depois volto pro vscode e dá certo pelo F5 (run).
Mas, realmente, o IOS sempre tem alguns probleminhas que aparecem e somem sem muita razão de acontecerem. 
Espero que isso te ajude também. Grande abraço.
